ConnectorNavisworks/ConnectorNavisworks/Bindings/ConnectorNavisworksBindings.Filters.cs

```diff
@@ -68,20 +66,31 @@ public override List<ISelectionFilter> GetSelectionFilters()
       filters.Add(savedViewsFilter);
     }
 
-    var clashPlugin = _doc.GetClash();
-    var clashTests = clashPlugin.TestsData;
-
-    var groupedClashResults = clashTests?.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+    try
+    {
+      var clashPlugin = _doc.GetClash();
+      var clashTests = clashPlugin?.TestsData;
 
-    if (groupedClashResults?.Count >= 0)
+      if (clashTests != null)
+      {
+        var groupedClashResults = clashTests.Tests.Select(GetClashTestResults).Where(x => x != null).ToList();
+
+        if (groupedClashResults.Count >= 0)
+        {
+          //  var clashReportFilter = new TreeSelectionFilter
+          //  {
+          //    Slug = "clashes", Name = "Clash Detective Results", Icon = "MessageAlert",
+          //    Description = "Select group clash test results.",
+          //    Values = groupedClashResults
+          //  };
+          //  filters.Add(clashReportFilter);
+        }
+        Console.WriteLine("Sending Clashes is not supported yet.");
+      }
+    }
+    catch (NullReferenceException)
     {
-      //  var clashReportFilter = new TreeSelectionFilter
-      //  {
-      //    Slug = "clashes", Name = "Clash Detective Results", Icon = "MessageAlert",
-      //    Description = "Select group clash test results.",
-      //    Values = groupedClashResults
-      //  };
-      //  filters.Add(clashReportFilter);
+      // ignore
```

Comment:
😨 So.... you are putting this here as belt and braces but it's going to swallow any NREs, if it is to catch unexpected NREs then we should at least be logging them with some consideration as to how verbose that might be

Reply:
removed